Sage/carbon
# docs(draggable): display expected prop types for props in docs

Currently id and children do not have prop types displayed in the props table in the docs for
draggable. This PR addresses this issue.

fixes #FE-4868

### Proposed behaviour

**Draggable**
- Remove duplicate `py` from prop table
- Give `id` and `children` valid prop types in the prop table

![Screenshot 2022-02-08 at 15 06 40](https://user-images.githubusercontent.com/56251247/153014811-775b22b4-28dd-4916-ab94-c6db86ff2c68.png)

**Flat Table Draggable Body**
- Give prop types comments so that the description is automatically populated in the docs

![image](https://user-images.githubusercontent.com/56251247/153381328-b7b6d913-143c-4ac5-8acb-be3a40ae490d.png)

### Current behaviour

**Draggable**
- Prop for `py` exists in both padding and prop tables
- `id` and `children` have prop types of `unknown` in our docs

![Screenshot 2022-02-08 at 15 07 11](https://user-images.githubusercontent.com/56251247/153014890-d5799cc7-6e16-41e7-b051-878229728496.png)

**Flat Table Draggable Body**
- props current do not have descriptions in the docs

![image](https://user-images.githubusercontent.com/56251247/153381090-6b655f1a-6a5e-4600-a792-55358edc7757.png)
### Checklist




#### QA


### Additional context

`required` flag in ArgsTable component does not put a red asterisk beside a required prop type. To compensate for this, I have added "This prop is required" to the prop type descriptions. 

### Testing instructions

- Check that the prop type descriptions for `DraggableItem` match that of the comments and PropTypes specified in the component file.

- Check that the descriptions for `FlatTableBodyDraggable` match that of the comments and PropTypes specified in the component file.
